CMakeLists.txt

```diff
@@ -83,23 +60,88 @@ INCLUDE_DIRECTORIES(
   ${EIGEN3_INCLUDE_DIR}
 )
 
-TARGET_LINK_LIBRARIES(${OM_WRAPPER_LIB}
-# debug ${OpenGL_LIBRARIES} optimized ${OpenGL_LIBRARIES}
-  debug ${OpenCV_LIBRARIES} optimized ${OpenCV_LIBRARIES}
-  debug ${ORB_SLAM2_LIB} optimized ${ORB_SLAM2_LIB}
-  debug ${DBOW2_LIB} optimized ${DBOW2_LIB}
-  debug ${G2O_LIB} debug ${G2O_LIB}
-)
-
-TARGET_LINK_LIBRARIES(${OM_WRAPPER_APP}
+################################
+# Define OpenMapper Library
+################################
+IF(BUILD_IOS_FW)
+  INCLUDE(cmake_modules/ios_10_3_config.cmake)
+  INCLUDE(cmake_modules/xcode_framework_config.cmake)
+
+  ADD_LIBRARY(${OM_WRAPPER_LIB} SHARED ${OM_LIB_SOURCES})
+  # Configure build targets for iOS
+  CONFIGURE_FOR_IOS_10_3(${OM_WRAPPER_LIB})
+  CONFIGURE_FOR_IOS_10_3(${ORB_SLAM2_LIB})
+  CONFIGURE_FOR_IOS_10_3(${DBOW2_LIB})
+  CONFIGURE_FOR_IOS_10_3(${G2O_LIB})
+
+  CONFIGURE_FOR_XCODE_FW(${OM_WRAPPER_LIB} "${CMAKE_CURRENT_LIST_DIR}/wrapper/ios/open_mapper.h")
+
+  TARGET_LINK_LIBRARIES(${OM_WRAPPER_LIB}
+    debug ${ORB_SLAM2_LIB} optimized ${ORB_SLAM2_LIB}
+    debug ${DBOW2_LIB} optimized ${DBOW2_LIB}
+    debug ${G2O_LIB} debug ${G2O_LIB}
```

Comment:
on mac, it was building before merging, but now it didnt either. maybe it was because at home i have g2o installed with homebrew :/ but then, travis passed also.. hmm